docs/howto/access-data.md

```diff
@@ -0,0 +1,45 @@
+### **Accessing the data**
+
+**Option 1: Using the data links**
+
+If we already have a workflow in place for downloading our data, we can use *earthaccess* as a search-only library and get HTTP links from our query results. This could be the case if our current workflow uses a different language and we only need the links as input.
+
+```python
+
+# if the data set is cloud hosted there will be S3 links available. The access parameter accepts "direct" or "external", direct access is only possible if you are in the us-west-2 region in the cloud.
+data_links = [granule.data_links(access="direct") for granule in results]
```

Comment:
I feel a little lost with this how-to coming right after "Authenticate with Earthdata Login". It may make more sense to move this after the Search how-to's so that `results` is more contextual. And/or adding a basic `results` code block above this.